eth/handler_eth_test.go

```diff
@@ -462,6 +463,84 @@ func testTransactionPropagation(t *testing.T, protocol uint) {
 	}
 }
 
+// Tests that transactions get propagated to all peers using TransactionMessages and not PooledTransactionHashesMsg
```

Comment:
could you add a `Quorum` tag? forgot to add this to detail that is a Quorum related logic